datasets/hlgd/README.md

```diff
@@ -0,0 +1,191 @@
+---
+annotations_creators:
+- crowdsourced
+extended:
+- original
```

Comment:
I think you should remove this part here
```suggestion
extended:
- original
```

Reply:
Alright I've removed it but I had build this YAML using this tool: https://huggingface.co/datasets/tagging/
Is it a problem of different versions of the YAML formats?

In any case, it seems to have solved the problem, so thank you for the help figuring it out!

Reply:
I use [this](https://github.com/huggingface/datasets-tagging) for dataset tagging